src/credential/index.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import * as credentialApi from './credential';
+import * as googleOauthAccessTokenApi from './google-oauth-access-token';
```

Comment:
`GoogleOAuthAccessToken` interface should also be exported.  But looks like it's part of the `admin` namespace today. So perhaps we should do it later once we start refactoring the top-level `admin` namespace.